models/articles.model.js

```diff
@@ -49,4 +49,28 @@ exports.selectAllArticles = () => {
     });
 };
 
+exports.selectAllCommentsByArticleId = (articleId) => {
+  const parsedArticleId = parseInt(articleId);
+  if(isNaN(parsedArticleId)) {
```

Comment:
You should have a psql error handler that is checking if the id is a number. There is no need for a custom error handler here.